README.md

```diff
@@ -21,10 +21,131 @@ It implements the following architecture:
+      
+      - Build (both release and debug) the projects inside the sdk using VS15: 
+        
+        '\LibOVR\Projects\Windows\VS2015\LibOVR.vcxproj'
+        
+        '\LibOVRKernel\Projects\Windows\VS2015\LibOVRKernel.vcxproj '
+      
+      - Add the following paths to the variables of robotology/YARP (because the yarpdev which gets the data from the sdk of the ovr and publishes in yarp framework, has dependencies on this library)
+          
+          
+              `LibOVR_LibOVRKernel_INCLUDE_DIR`:'<path to codes workspace>/OculusSDK/LibOVRKernel/Src'
+              
+              `LibOVR_LibOVRKernel_LIBRARY_DEBUG`:'<path to codes workspace>/OculusSDK/LibOVRKernel/Lib/Windows/x64/Debug/VS2015/LibOVRKernel.lib'
+
+              `LibOVR_LibOVRKernel_LIBRARY_RELEASE`: '<path to codes workspace>/OculusSDK/LibOVRKernel/Lib/Windows/x64/Release/VS2015/LibOVRKernel.lib'
+
+              `LibOVR_LibOVR_Extras_INCLUDE_DIR`: '<path to codes workspace>/OculusSDK/LibOVR/Include/Extras'
+
+              `LibOVR_LibOVR_INCLUDE_DIR:PATH`: '<path to codes workspace>/OculusSDK/LibOVR/Include'
+
+              `LibOVR_LibOVR_LIBRARY_DEBUG`: '<path to codes workspace>/OculusSDK/LibOVR/Lib/Windows/x64/Debug/VS2015/LibOVR.lib'
+
+              `LibOVR_LibOVR_LIBRARY_RELEASE`: '<path to codes workspace>/OculusSDK/LibOVR/Lib/Windows/x64/Release/VS2015/LibOVR.lib'
+
+              `YARP_USE_LOBOVR`: check the box
+
+      - Add the following varibales to the Rapid Evironment Edittor:
+             
+              `OculusSDK_ROOT`= '<path to codes workspace>\OculusSDK' (variable type is String)'
+              `Path`= '<path to codes workspace>\OculusSDK\3rdParty\Windows Kits\8.1\Redist\D3D\x64'
+              
+             
+      - Configure, generate the cmake (cmake gui --> robotology yarp) and build the yarp using vs15.
+
+
+6. **Cyberith SDK:**   To allow the virtualizer module to capture the operator data, the virtualizer needs two modules to download: the CybSDK_app (used for the calibration!) and the CybSDK:
+
+   1. `CybSDK_app`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to 'Tools' tab, download the 'Virtualizer Control Panel'. Copy the download application to the workspace. Everytime start using the virtualizer, please calibrate it using this application.
+     
+   2. `CybSDK`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to 'SDK' tab, download the 'C++ SDK for Windows (Developed for: Visual Studio 2015 Community)'. Copy the download application to the workspace. in theory, we should build it in our Windows machine, but time being it is already built (there exists the .dll file) and is working, so you do not need to build it.
+   
+      - Add the following variables to the Rapid Evironment Editor:
```

Comment:
Like before you can avoid mentioning `Rapid Evironment Editor`

Reply:
Or if you want follow @traversaro suggestion https://github.com/robotology/walking-teleoperation/pull/9#discussion_r243104349